styles.css

```diff
@@ -1,785 +1,862 @@
+            font-weight: 500;
+            font-size: 1.25vw;
+            line-height: 4vw;
+            letter-spacing: 0.03em;
+            color: #6070FF;
+            flex: none;
+            order: 0;
+            flex-grow: 0;
+            border: none;
+            padding-left: 0.5vw;
+          }
+          .rail{
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            padding: 4px 12px;
+            width: 69.2%;
+            height: 2.5vw;
+            background: #EBEBFF;
+            border-radius: 8px;
+            flex: none;
+            order: 1;
+            flex-grow: 0;
+          }
+          .railbut{
+            width: 75%;
+            height: 4vw;
+            font-family: 'Poppins', sans-serif;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 1.25vw;
+            line-height: 4vw;
+            letter-spacing: 0.03em;
+            color: #6070FF;
+            flex: none;
+            order: 0;
+            flex-grow: 0;
+            border: none;
+            padding-left: 0.5vw;
+          }
+          .see{
+            width: 15.5vw;
+            height: 6vw;
+            font-family: 'Poppins', sans-serif;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 1.75vw;
+            line-height: 8vw;
+            display: flex;
+            align-items: center;
+            text-align: center;
+            padding-left: 2vw;
+            letter-spacing: 0.03em;
+            color: #396DF2;
+            flex: none;
+            order: 3;
+            flex-grow: 0;
+            border: #396DF2 solid 2px;
+            background-color: #FFFFFF;
+            border-radius: 8px;
+            margin-left: 5vw;
+            margin-top: 2vw;
+            cursor: pointer;
+          }
+          .see:hover{background-color: #396DF2; border:  #FFFFFF solid 2px;color:  #FFFFFF;}
+          .see:active{background-color: #FFFFFF;color: #000000;border:  #FFFFFF solid 2px;}
+          .partone11{
+            width: 50vw;
+            height: 100%;
+            background-image: url("Portfolio2.png");
+            background-size: 45vw 100%;
+            background-repeat: no-repeat;
+        }
+        .parttwo1{
+            width: 50vw;
+            height: 100%;
+            background-image: url("Portfolio1.png");
+            background-size: 45vw 100%;
+            background-repeat: no-repeat;
+        }
+        .parttwo2{
+            width: 50vw;
+            height: 100%;
+            background-image: url("Portfolio3.png");
+            background-size: 45vw 100%;
+            background-repeat: no-repeat;
+        }
+        .aboutpage{
+            display: flex;
+            flex-direction: row;
+            flex-wrap: nowrap;
+            width: 100vw;
+            height: 100%;
+            background: white;
+            border-bottom-right-radius: 120px;
+        }

```

Comment:
- [ ] This is close but as you can see from the Figma design the content in the about me section on the left takes 40% and on the right one the remaining 60% of the page's width in your design both seem to take equal width so please update this.
- Figma design:
  ![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/f77e5d1e-3fd1-4fa0-a5dc-66c9332df64b)
- Your design:
  ![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/6ba3775b-52fb-4991-aad8-a40a3140e020)